### PR TITLE
release-6.5.0: update the release note introduction 

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v7.3 (TiDB 7.3 versions)
- [x] v7.2 (TiDB 7.2 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v7.0 (TiDB 7.0 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by samhld — COMMENTED** [review]
LGTM!

File: releases/release-6.5.0.md [hunk 1]
```diff
@@ -13,7 +13,14 @@ Quick access: [Quick start](https://docs.pingcap.com/tidb/v6.5/quick-start-with-
 
 TiDB 6.5.0 is a Long-Term Support Release (LTS).
 
-Compared with the previous LTS 6.1.0, 6.5.0 not only includes new features, improvements, and bug fixes released in [6.2.0-DMR](/releases/release-6.2.0.md), [6.3.0-DMR](/releases/release-6.3.0.md), [6.4.0-DMR](/releases/release-6.4.0.md), but also introduces the following key features and improvements:
+Compared with TiDB [6.4.0-DMR](/releases/release-6.4.0.md), TiDB 6.5.0 introduces the following key features and improvements:
+
+> **Tip**
```
Comment (lilin90) [inline releases/release-6.5.0.md]:
```suggestion
> **Tip:**
```

File: releases/release-6.5.0.md [hunk 2]
```diff
@@ -13,7 +13,14 @@ Quick access: [Quick start](https://docs.pingcap.com/tidb/v6.5/quick-start-with-
...
+>
+> Compared with the previous LTS 6.1.0, TiDB 6.5.0 also include new features, improvements, and bug fixes released in [6.2.0-DMR](/releases/release-6.2.0.md), [6.3.0-DMR](/releases/release-6.3.0.md), [6.4.0-DMR](/releases/release-6.4.0.md).
```
Comment (qiancai) [inline releases/release-6.5.0.md:20]:
```suggestion
> Compared with the previous LTS 6.1.0, TiDB 6.5.0 also include new features, improvements, and bug fixes released in [6.2.0-DMR](/releases/release-6.2.0.md), [6.3.0-DMR](/releases/release-6.3.0.md), and [6.4.0-DMR](/releases/release-6.4.0.md).
```

File: releases/release-6.5.0.md [hunk 3]
```diff
@@ -13,7 +13,14 @@ Quick access: [Quick start](https://docs.pingcap.com/tidb/v6.5/quick-start-with-
...
 
-Compared with the previous LTS 6.1.0, 6.5.0 not only includes new features, improvements, and bug fixes released in [6.2.0-DMR](/releases/release-6.2.0.md), [6.3.0-DMR](/releases/release-6.3.0.md), [6.4.0-DMR](/releases/release-6.4.0.md), but also introduces the following key features and improvements:
+Compared with TiDB [6.4.0-DMR](/releases/release-6.4.0.md), TiDB 6.5.0 introduces the following key features and improvements:
+
+> **Tip**
+>
+> Compared with the previous LTS 6.1.0, TiDB 6.5.0 also include new features, improvements, and bug fixes released in [6.2.0-DMR](/releases/release-6.2.0.md), [6.3.0-DMR](/releases/release-6.3.0.md), and [6.4.0-DMR](/releases/release-6.4.0.md).
+>
+> - To get a full list of changes between the LTS 6.1.0 and 6.5.0, besides this release notes, also see [6.2.0-DMR release notes](/releases/release-6.2.0.md), [6.3.0-DMR release notes](/releases/release-6.3.0.md), and [6.4.0-DMR release notes](/releases/release-6.4.0.md).
+> - To have a quick comparison of key features between LTS 6.1.0 and 6.5.0, you can check the `v6.1` and `v6.5` columns in [TiDB features](/basic-features.md).
```
Comment (qiancai) [inline releases/release-6.5.0.md:23]:
```suggestion
    > **Tip**
    >
    > Compared with the previous LTS 6.1.0, TiDB 6.5.0 also include new features, improvements, and bug fixes released in [6.2.0-DMR](/releases/release-6.2.0.md), [6.3.0-DMR](/releases/release-6.3.0.md), and [6.4.0-DMR](/releases/release-6.4.0.md).
    >
    > - To get a full list of changes between the LTS 6.1.0 and 6.5.0, besides this release notes, also see [6.2.0-DMR release notes](/releases/release-6.2.0.md), [6.3.0-DMR release notes](/releases/release-6.3.0.md), and [6.4.0-DMR release notes](/releases/release-6.4.0.md).
    > - To have a quick comparison of key features between LTS 6.1.0 and 6.5.0, you can check the `v6.1` and `v6.5` columns in [TiDB features](/basic-features.md).
```

File: releases/release-6.5.0.md [hunk 4]
```diff
@@ -13,7 +13,14 @@ Quick access: [Quick start](https://docs.pingcap.com/tidb/v6.5/quick-start-with-
...
 
-Compared with the previous LTS 6.1.0, 6.5.0 not only includes new features, improvements, and bug fixes released in [6.2.0-DMR](/releases/release-6.2.0.md), [6.3.0-DMR](/releases/release-6.3.0.md), [6.4.0-DMR](/releases/release-6.4.0.md), but also introduces the following key features and improvements:
+Compared with TiDB [6.4.0-DMR](/releases/release-6.4.0.md), TiDB 6.5.0 introduces the following key features and improvements:
+
+    > **Tip**
+    >
+    > Compared with the previous LTS 6.1.0, TiDB 6.5.0 also include new features, improvements, and bug fixes released in [6.2.0-DMR](/releases/release-6.2.0.md), [6.3.0-DMR](/releases/release-6.3.0.md), and [6.4.0-DMR](/releases/release-6.4.0.md).
+    >
+    > - To get a full list of changes between the LTS 6.1.0 and 6.5.0, besides this release notes, also see [6.2.0-DMR release notes](/releases/release-6.2.0.md), [6.3.0-DMR release notes](/releases/release-6.3.0.md), and [6.4.0-DMR release notes](/releases/release-6.4.0.md).
+    > - To have a quick comparison of key features between LTS 6.1.0 and 6.5.0, you can check the `v6.1` and `v6.5` columns in [TiDB features](/basic-features.md).
```
Comment (qiancai) [inline releases/release-6.5.0.md:23]:
```suggestion
> **Tip**
>
> Compared with the previous LTS 6.1.0, TiDB 6.5.0 also include new features, improvements, and bug fixes released in [6.2.0-DMR](/releases/release-6.2.0.md), [6.3.0-DMR](/releases/release-6.3.0.md), and [6.4.0-DMR](/releases/release-6.4.0.md).
>
> - To get a full list of changes between the LTS 6.1.0 and 6.5.0, besides this release notes, also see [6.2.0-DMR release notes](/releases/release-6.2.0.md), [6.3.0-DMR release notes](/releases/release-6.3.0.md), and [6.4.0-DMR release notes](/releases/release-6.4.0.md).
> - To have a quick comparison of key features between LTS 6.1.0 and 6.5.0, you can check the `v6.1` and `v6.5` columns in [TiDB features](/basic-features.md).
```

File: releases/release-6.5.0.md [hunk 5]
```diff
@@ -13,7 +13,14 @@ Quick access: [Quick start](https://docs.pingcap.com/tidb/v6.5/quick-start-with-
...
 TiDB 6.5.0 is a Long-Term Support Release (LTS).
 
-Compared with the previous LTS 6.1.0, 6.5.0 not only includes new features, improvements, and bug fixes released in [6.2.0-DMR](/releases/release-6.2.0.md), [6.3.0-DMR](/releases/release-6.3.0.md), [6.4.0-DMR](/releases/release-6.4.0.md), but also introduces the following key features and improvements:
+Compared with TiDB [6.4.0-DMR](/releases/release-6.4.0.md), TiDB 6.5.0 introduces the following key features and improvements:
+
+> **Tip:**
+>
+> Compared with the previous LTS 6.1.0, TiDB 6.5.0 also include new features, improvements, and bug fixes released in [6.2.0-DMR](/releases/release-6.2.0.md), [6.3.0-DMR](/releases/release-6.3.0.md), and [6.4.0-DMR](/releases/release-6.4.0.md).
+>
+> - To get a full list of changes between the LTS 6.1.0 and 6.5.0, besides this release notes, also see [6.2.0-DMR release notes](/releases/release-6.2.0.md), [6.3.0-DMR release notes](/releases/release-6.3.0.md), and [6.4.0-DMR release notes](/releases/release-6.4.0.md).
```
Comment (lilin90) [inline releases/release-6.5.0.md:22]:
```suggestion
> - To get a full list of changes between the 6.1.0 LTS and 6.5.0 LTS versions, in addition to this release notes, also see [6.2.0-DMR release notes](/releases/release-6.2.0.md), [6.3.0-DMR release notes](/releases/release-6.3.0.md), and [6.4.0-DMR release notes](/releases/release-6.4.0.md).
```

File: releases/release-6.5.0.md [hunk 6]
```diff
@@ -13,7 +13,14 @@ Quick access: [Quick start](https://docs.pingcap.com/tidb/v6.5/quick-start-with-
...
+> - To have a quick comparison of key features between LTS 6.1.0 and 6.5.0, you can check the `v6.1` and `v6.5` columns in [TiDB features](/basic-features.md).
```
Comment (lilin90) [inline releases/release-6.5.0.md:23]: How about using a style similar to that in https://github.com/pingcap/docs#documentation-versions
```suggestion
> - To have a quick comparison of key features between the 6.1.0 LTS and 6.5.0 LTS versions, you can check the `v6.1` and `v6.5` columns in [TiDB features](/basic-features.md).
```

File: releases/release-6.5.0.md [hunk 7]
```diff
@@ -13,7 +13,14 @@ Quick access: [Quick start](https://docs.pingcap.com/tidb/v6.5/quick-start-with-
 
 TiDB 6.5.0 is a Long-Term Support Release (LTS).
 
-Compared with the previous LTS 6.1.0, 6.5.0 not only includes new features, improvements, and bug fixes released in [6.2.0-DMR](/releases/release-6.2.0.md), [6.3.0-DMR](/releases/release-6.3.0.md), [6.4.0-DMR](/releases/release-6.4.0.md), but also introduces the following key features and improvements:
+Compared with TiDB [6.4.0-DMR](/releases/release-6.4.0.md), TiDB 6.5.0 introduces the following key features and improvements:
+
+> **Tip:**
+>
+> Compared with the previous LTS 6.1.0, TiDB 6.5.0 also include new features, improvements, and bug fixes released in [6.2.0-DMR](/releases/release-6.2.0.md), [6.3.0-DMR](/releases/release-6.3.0.md), and [6.4.0-DMR](/releases/release-6.4.0.md).
```
Comment (qiancai) [inline releases/release-6.5.0.md:20]:
```suggestion
> Compared with the previous LTS 6.1.0, TiDB 6.5.0 also includes new features, improvements, and bug fixes released in [6.2.0-DMR](/releases/release-6.2.0.md), [6.3.0-DMR](/releases/release-6.3.0.md), and [6.4.0-DMR](/releases/release-6.4.0.md).
```

**Pull request merged**